cypress/support/graphql-utils.ts

```
@@ -0,0 +1,147 @@
+/**
+ * GraphQL interception utilities for Cypress
+ *
+ * Provides typed, reusable helpers to intercept GraphQL operations
+ * by operationName for success and error scenarios.
+ */
+
+/**
+ * Type for GraphQL response handler
+ * - string: fixture path (e.g., 'api/graphql/organizations.success.json')
+ * - Record<string, unknown>: inline response object
```

ESLint

[error] tsdoc-html-tag-missing-greater-than: The HTML tag has invalid syntax: Expecting an attribute or ">" or "/>"

(tsdoc/syntax)